### PR TITLE
Rework Build

### PR DESCRIPTION
**Pull request by Odomontois**

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -1,27 +1,18 @@
-name-template: '$NEXT_PATCH_VERSION'
-tag-template: '$NEXT_PATCH_VERSION'
+name-template: 'Derevo $NEXT_MINOR_VERSION'
```
Comment (catostrophe) [inline .github/release-drafter.yml:1]: fix please

File: .github/mergify.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+pull_request_rules:
```
Comment (catostrophe) [inline .github/mergify.yml:1]: Update this file from https://github.com/tofu-tf/derevo/pull/297

File: build.sbt [hunk 3]
```diff
@@ -27,55 +20,63 @@ lazy val defaultSettings = Seq(
...
 
-lazy val higherKindCore = project settings (
-  defaultSettings,
-  publishName := "core-higher-kind",
-  libraryDependencies ++= Seq(catsCore, catsFree, catsTagless),
-)
+lazy val higherKindCore = project
+  .in(file("modules/higherKindCore"))
+  .settings(
+    defaultSettings,
```
Comment (catostrophe) [inline build.sbt:28]: each module should be provided with publish/noPublish settings

File: ci-release.sbt [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ThisBuild / developers := List(
+  Developer("catostrophe", "λoλcat", "catostrophe@pm.me", url("https://github.com/catostrophe")),
+  Developer("danslapman", "Daniil Smirnov", "danslapman@gmail.com", url("https://github.com/danslapman")),
+  Developer("odomontois", "Oleg Nizhnik", "odomontois@gmail.com", url("https://github.com/odomontois")),
+  Developer("oskin1", "Ilya Oskin", "ilya.arcadich@gmail.com", url("https://github.com/oskin1")),
+)
+
+ThisBuild / homepage := Some(url("https://github.com/tofu-tf/tofu"))
+
+ThisBuild / description := "Opinionated set of tools for functional programming in Scala"
```
Comment (catostrophe) [inline ci-release.sbt:46]: where's organization?

File: ci-release.sbt [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/TinkoffCreditSystems/tofu"),
```
Comment (catostrophe) [inline ci-release.sbt:50]: Tinkoff -> tofu

**Review by catostrophe — CHANGES_REQUESTED** [review]

File: .github/release-drafter.yml (hunk 1 above)
Comment (Odomontois) [reply .github/release-drafter.yml:1]: OMG, I was sue i changed this

File: build.sbt (hunk 3 above)
Comment (catostrophe) [reply build.sbt:28]: I see, only docs don't get published. and what about the root itself?

Comment (Odomontois) [reply build.sbt:28]: We are usually publishing it under `tofu`, it's like swissknife

File: .github/mergify.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  - name: Request reviews and label Steward's PRs
+    conditions:
+      - author=scala-steward
+    actions:
+      request_reviews:
+        users: [ Odomontois, catostrophe ]
```
Comment (catostrophe) [inline .github/mergify.yml:7]: please add more reviewers

**Review by catostrophe — APPROVED** [review]

**Pull request merged**